codecs/jxl/Makefile

```diff
@@ -78,9 +78,13 @@ $(CODEC_MT_SIMD_BUILD_DIR)/Makefile: CXXFLAGS+=-msimd128
 	-B $(@D) \
 	$(<D)
 
-$(CODEC_DIR)/CMakeLists.txt:
-	mkdir -p $(@D)
-	git clone $(CODEC_URL) --recursive -j`nproc` --depth 1 --branch $(CODEC_VERSION) $(@D)
+$(CODEC_DIR)/CMakeLists.txt: 
+	rm -rf $(@D)
+	git init $(@D)
+	git -C $(@D) remote add origin $(CODEC_URL)
```

Comment:
This line is unnecessary, you can just `fetch $(CODEC_VERSION)` below.

Reply:
Nice!